extensions/platform-http/component/src/main/java/org/apache/camel/component/http/server/PlatformHttpConsumer.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.http.server;
+
+import java.util.Set;
+
+import org.apache.camel.Endpoint;
+import org.apache.camel.Exchange;
+import org.apache.camel.ExchangePattern;
+import org.apache.camel.Processor;
+import org.apache.camel.component.http.server.spi.HttpEndpointSpec;
+import org.apache.camel.component.http.server.spi.Method;
+import org.apache.camel.component.http.server.spi.PlatformHttpEngine;
+import org.apache.camel.spi.HeaderFilterStrategy;
+import org.apache.camel.support.DefaultConsumer;
+
+public class PlatformHttpConsumer extends DefaultConsumer implements HttpEndpointSpec {
```

Comment:
This class may not bee needed if the creation of the consumer is delegated to the engine bits.

Reply:
`HttpEndpointSpec` is gone, but `QuarkusPlatformHttpConsumer` is still needed IMO.